src/locales/es_ES/es_ES.json

```diff
@@ -137,6 +137,16 @@
         "detail": {
             "folded_doc_front": "Asegúrese que su documento este completmente abierto (debe incluir su foto)"
         },
+        "header": {
+            "passport": "Passport photo page",
+            "license_front": "Front of driver’s license",
+            "license_back": "Back of driver’s license",
+            "id_front": "Front of ID card",
+            "id_back": "Back of ID card",
+            "permit_front": "Front of residence permit card",
+            "permit_back": "Back of residence permit card",
+            "progress": "Capturing your document..."
+        },
```

Comment:
Is this feature going to be in English only? Is this documented in the README?

Reply:
This will be documented in `MIGRATION.md`, but currently, we have plenty of keys waiting for release in the next version after `6.9.0`, I'll wait after that to avoid unnecessary conflicts.

Reply:
Ok, I will leave this conversation unresolved so you can remember to add details to `MIGRATION.md` before merging

Reply:
Since `6.9.0` copy updates has/should be merged in already, should `MIGRATION.md` be updated in this PR also now?